src/mantine-core/src/Button/Button.styles.ts

```diff
@@ -135,6 +135,15 @@ export default createStyles(
 
       '&:active': theme.activeStyles,
 
+      '&:disabled': {
```

Comment:
Should be `&:disabled, &[data-disabled]` to avoid styles duplication

Reply:
@rtivital sure I will add this to ActionIcon too. 

Should I be adding this as a new style (as in this screenshot) or simply updating the existing style (the one marked 2 in screenshot): 
<img width="1776" alt="image" src="https://user-images.githubusercontent.com/8950327/190126568-661dd977-8112-47b9-b663-aa66cf8d0294.png">

Reply:
Update the existing one